webAO/client.js

```diff
@@ -887,15 +889,18 @@ class Client extends EventEmitter {
   async handleSC(args) {
     const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
 
-    // Add this so people can see characters loading on the screen.
-    document.getElementById('client_loading').style.display = 'none';
-    document.getElementById('client_charselect').style.display = 'block';
+    if (mode === 'watch') {		// Spectators don't need to pick a character
```

Comment:
Pressing "Or spectate" causes the same issue where it shows the "Unable to connect"
![image](https://user-images.githubusercontent.com/36182383/158710847-e57eaf22-1318-43d1-bccd-fab83b7ccef4.png)

Reply:
[3ee77d0](https://github.com/AttorneyOnline/webAO/pull/114/commits/3ee77d0312931f3ebaae8ef83ad533159c4ff546)